src/lib/operators/jit_operator/jit_types.cpp

```diff
@@ -0,0 +1,65 @@
+#include "jit_types.hpp"
+
+namespace opossum {
+
+template <>
+JitDataType jit_data_type<uint8_t>() {
+  return JitDataType::Bool;
```

Comment:
As mentioned above, we aim to have only one place where new types are defined. Any way you could get that information from all_type_variant.hpp? Maybe @mjendruk can chime in on this, because he came up with `DATA_TYPE_INFO`.

Reply:
Found a solution. Thanks, @mjendruk!